### ISSUE TITLE
Heroku Deploy Erroring After Several Minutes

### ISSUE THREAD

**Issue by MarcDonald** [MarcDonald/twitch-spotify-request-bot#18]
When the app is deployed out to Heroku, it works fine for a while, but then shuts down seemingly randomly after a few minutes. This occurs when the app is left idle AND when the app is in constant use. Issue first identified by [ssbPleb ](https://twitter.com/ssbPleb) on Twitter and I can recreate it on my own Heroku account with a fresh app created.

This is an example of the logs when the error occurs.
![Example of logs that print when the error occurs](https://user-images.githubusercontent.com/19517260/145294888-4c491a46-0fe6-475f-9c6b-96318e5288b7.png)

When Heroku restarts the app, there seems to be two instances which work in parallel for a short amount of time, resulting in messages being processed twice.

![Messages being processed twice](https://user-images.githubusercontent.com/19517260/145301405-128b8b5a-5d81-49d8-92e5-78014210c008.png)

After a minute or so Heroku fully shuts down both of the instances.

![Error logs when Heroku shuts down the instances](https://user-images.githubusercontent.com/19517260/145297701-cc05d1e0-521f-4d3c-8094-9e5a4f98ed9f.png)

Heroku on the free tier does shut down instances after 30 minutes of sitting idle, however this error occurs after only a few minutes and even occurs when the instance is not idle.

**Comment by IvanGodinez21** [comment]
I discovered that when you click on 'Open app' it triggers 'web' in heroku, so it will run npm start again, this will happen every time that the user enters the auth link displayed in the console or clicks on 'Open app' (wich are the same) as I understand.
More information in the following links:

https://devcenter.heroku.com/articles/dynos#dyno-configurations
https://devcenter.heroku.com/articles/dyno-types
https://stackoverflow.com/a/38965136

I tried enabling only the 'worker' and disabling 'web' but when I try to authenticate by pressing 'Open app', it displays in the console:

```
heroku[router]: at=error code=H21 desc="Backend connection refused" method=GET path="/" host=twitch-spotify-request-bot.herokuapp.com request_id=eae730a9-eca3-4096-9b94-0daad5803b61 fwd="177.227.33.136" dyno=web.1 connect=0ms service= status=503 bytes= protocol=https
```
So you are forced to allow the 'web' to be enabled. If you enable both options, it's still not working.

The solution for this should be one of the following:
- Get the 'worker' to work (hehe) with express without using the 'web' (Basically, investigate a way to run express in Heroku without using 'web')
- Change the host service from Heroku to another (I heard about Vercel, Netlify, DigitalOcean, but I don't know If those have a free plans and easy usage like Heroku)
